cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -190,6 +190,27 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
               }
             });
           });
+          it('should have subtitles set to enabled by default', () => {
+            cy.window().then(win => {
+              const media = getBlockData('video', win.SIMORGH_DATA.pageData);
+              if (media && media.type === 'video') {
+                cy.get('div[class^="StyledVideoContainer"]').then(() => {
+                  cy.get('iframe[class^="StyledIframe"]').then($iframe => {
+                    cy.wrap($iframe.prop('contentWindow'), {
+                      // `timeout` only applies to the methods chained below.
+                      // `its()` benefits from this, and will wait up to 8s
+                      // for the mediaPlayer instance to become available.
+                      timeout: 8000,
```

Comment:
Is the timeout of 8 seconds necessary? I ask this since we will be running these tests for all articles and it'll slow down our end to end tests quite a bit - the cron job will have this running for each service. This could be over 5 minutes just for this single test.

Reply:
From my understanding, it's waiting for the instance to be available, rather than pausing the test entirely, which is what `cy.wait` would do. It should also be worth mentioning that when I've run this test, luckily, it doesn't use the entire `8000` provided.